care/facility/api/serializers/patient_consultation.py

```diff
@@ -192,6 +196,34 @@ def validate_bed_number(self, bed_number):
             bed_number = None
         return bed_number
 
+    def get_mews_field(self, consultation):
+        current_time = localtime(now())
+        past_30_minutes = current_time - timedelta(minutes=30)
+        recentDailyArray = DailyRound.objects.filter(rounds_type = DailyRound.RoundsType.NORMAL.value ,consultation = consultation ,modified_date__gte=past_30_minutes, modified_date__lte=current_time).order_by("-modified_date")
```

Comment:
```suggestion
        recentDailyArray = DailyRound.objects.filter(rounds_type = DailyRound.RoundsType.NORMAL.value ,consultation = consultation ,modified_date__gte=past_30_minutes, modified_date__lte=current_time).order_by("-modified_date").only(mews_field_data.keys())
```
reduce the number of fields returned by the qs


```suggestion
        recentDailyArray = DailyRound.objects.filter(rounds_type = DailyRound.RoundsType.NORMAL.value ,consultation = consultation ,modified_date__gte=past_30_minutes, modified_date__lte=current_time).order_by("-modified_date").only(mews_field_data.keys())[:5]
```
and you can directly limit the qs to 5 results, no need to manually keep track of the count

Reply:
we need all logs in last 30 mins if there would be more than 5 logs and one parameter if updated in the 6th one, we wouldn't get it? I'll update the only(mews_field_data.keys())

Reply:
I added this  only(*(list(mews_field_data.keys()))), because only(mews_field_data.keys()) was throwing error,

Reply:
![image](https://github.com/coronasafe/care/assets/91010142/c419b06e-800e-41f5-820e-232fbd9f2e6d)
I'm using here count variable to check if at any point the all required data is got updated the loop will break without traversing furthure and will return the value

Reply:
please verify we need data from last 30 min or only last 5 logs, I'll change accordingly

Reply:
@sainak the issue states about 5 fields and not the last 5 rows.

Reply:
okay